include/SFML/Graphics/Text.hpp

```diff
@@ -299,14 +375,45 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     void ensureGeometryUpdate() const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Adds a line to the vertices
+    ///
+    /// This used for adding strikethrough and underline lines
+    ///
+    /// \param position         Width and top of the line
```

Comment:
"Position" sounds a bit misleading if you're actually referring to a (width, top) coordinate pair. Is there no better term?